.pylintrc

```diff
@@ -2,4 +2,7 @@
 
 # List of members which are set dynamically and missed by Pylint inference
 # system, and so shouldn't trigger E1101 when accessed.
-generated-members=numpy.*, torch.*
\ No newline at end of file
+generated-members=numpy.*, torch.*
+
+[MESSAGES CONTROL]
+disable=not-callable
```

Comment:
What is the reason for bypassing this check?
I hardly believe it was necessary and there are ways to make it work without hardcoding an explicit bypass

Reply:
it was giving me errors when i was calling text encoder, generator, discriminator, basically any pytorch module. Pytorch modules are callable but idk why it was showing that error, tried to Google it but didn't find any solution, if its possible to fix it without bypass can do it, let me know how.